src/test/java/org/minions/devfund/richard/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,108 @@
+package org.minions.devfund.richard.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test Class.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private static final String MOVIE_ONE = "Movie1";
+    private static final String MOVIE_TWO = "Movie2";
+    private static final double TEN = 10.0;
+    private static final double TWENTY = 20.0;
+
+    /**
+     * Before.
+     */
+    @Before
+    public void setUp() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Test.
+     */
+    @Test
+    public void testGetBestActor() {
+        movieDatabase.addMovie(MOVIE_ONE, new String[]{"Actor1", "Actor2"});
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor2" 14 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor1" 18 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)